### PR TITLE
New: Add Catalan Language - Lang 39

### LINKED ISSUE #6257: Catalan language is not in the list
**Is your feature request related to a problem? Please describe.**
When I want to set Catalan language to a movie I discover that this language is not in the list

**Describe the solution you'd like**
"Catalan" language appear should be in the list

[AB#611](https://dev.azure.com/Servarr/7ab38f4e-5a57-4d70-84f4-94dd9bc5d6df/_workitems/edit/611)

### PR DESCRIPTION
**Pull request by dtalens**
Add Catalan into language list

#### Database Migration
NO

#### Description
Add Catalan into language list. The translation using weblate is 100%. Translations update from Servarr Weblate #7415 

#### Todos
- [ ] Tests
- [ ] Translation Keys (./src/NzbDrone.Core/Localization/Core/en.json)
- [ ] [Wiki Updates](https://wiki.servarr.com)

#### Issues Fixed or Closed by this PR

* Fixes #6257

### CODE REVIEW

**Comment by bakerboy448** [comment]
- Lang 36 is reserved for https://github.com/Radarr/Radarr/pull/6997
- Lang 37 is reserved for https://github.com/Radarr/Radarr/pull/6415
- Lang 35 and 38 are reserved for https://github.com/Radarr/Radarr/pull/6522

This would need to be Lang 39

**Comment by Qstick** [comment]
@dtalens Is the intention here to support for UI translations, release parsing or both?

**Comment by dtalens** [comment]
> @dtalens Is the intention here to support for UI translations, release parsing or both?

Both

**Review by RobinDadswell — CHANGES_REQUESTED** [review]
Please don't use merge commits - only ever rebase

File: src/NzbDrone.Core/Parser/LanguageParser.cs [hunk 1]
```diff
@@ -27,7 +27,8 @@ public static class LanguageParser
                                                                             (?<hebrew>\bHebDub\b)|
                                                                             (?<polish>\b(?:PL\W?DUB|DUB\W?PL|LEK\W?PL|PL\W?LEK)\b)|
                                                                             (?<chinese>\[(?:CH[ST]|BIG5|GB)\]|简|繁|字幕)|
-                                                                            (?<ukrainian>(?:(?:\dx)?UKR))",
+                                                                            (?<ukrainian>(?:(?:\dx)?UKR))|
+                                                                            (?<catalan>\CA\b)",
```
Comment (bakerboy448) [inline src/NzbDrone.Core/Parser/LanguageParser.cs:31]: `\C` doesn't make sense?
Why escape it?

**Review by bakerboy448 — CHANGES_REQUESTED** [review]
Also needs to be reset to drop the merge commit and rebased instead as robin noted

**Comment by dtalens** [comment]
> Also needs to be reset to drop the merge commit and rebased instead as robin noted

Please, how should I do rebase? Thanks.

**Comment by bakerboy448** [comment]
> 
> 
> 
> 
> 
> 
> > Also needs to be reset to drop the merge commit and rebased instead as robin noted
> 
> 
> 
> Please, how should I do rebase? Thanks.

https://stackoverflow.com/questions/17577409/git-remove-merge-commit-from-history


Or reset but leave the changes stages back on upstream develop and then recommit your changes

**Pull request closed without merging**